Hexagon.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Drawing;
+
+namespace Lab_1_1
+{
+    class Hexagon: Figure
+    {
+        int x1, y1, x2, y2, x3, y3, x4, y4, x5, y5, x6, y6;
```

Comment:
if you use Points it's more accurate to store Point collection than this huge amount of coordinates that you don't need in future

Reply:
You can also take out this collection into Figure class